content/posts/numpy/numpy2/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "18 years in the Making: NumPy 2.0 Released"
+date: 2024-06-16
+draft: false
+description: "
+18 years in the Making: NumPy 2.0 Released
+"
+tags: ["News", "numpy"]
+displayInList: true
+author: ["NumPy Developers"]
+---
+
+After 18 years since the release of NumPy 1.0, we are thrilled to announce the
+launch of NumPy 2.0! This major release marks a significant milestone in the
+evolution of NumPy, bringing a wealth of enhancements and improvements to users
+and unlocking future developments.
+
+NumPy has improved and evolved over the past 18 years and many old releases had
+significant performance, usability, and consistency improvements.
+Our approach for a long time has been to make incremental changes while
+carefully managing backward compatibility. This approach finally had to come to
+an end, for this one-off major release, with the significant improvements to
+NumPy's type system. The type system is fundamental to NumPy, and major
+behavioral changes could not be made incrementally, since mixing two different
+type systems would be a recipe for disaster.
+
+The journey to an actual 2.0 release has been long, and it was difficult to
+build the necessary momentum. In part this may be because for a time the idea
+of doing a NumPy 2.0 release among the NumPy developers required a revolutionary change with
+rewrites of significant key pieces in the code base. Many of these rewrites and
```

Comment:
```suggestion
build the necessary momentum. In part, this may be because, for a time, the NumPy developers associated a NumPy 2.0 release with nothing less than a revolutionary 
rewrite of significant key pieces in the code base. Many of these rewrites and
```

Hope I'm capturing this correctly, but the sentence as is is a bit unclear.

Reply:
This does read better, and I think captures the intent well.

Reply:
Can also probably cut "In part, " to simplify the sentence further.